db.postgres.sql

```diff
@@ -0,0 +1,142 @@
+-- MySQL dump 10.19  Distrib 10.3.28-MariaDB, for Linux (x86_64)
```

Comment:
this is for postgres but mentions mysql/mariadb in the comment. a bit confusing.